.ci/generate_bear_requirements.py

```diff
@@ -168,6 +171,18 @@ def get_pip_requirements(requirements):
 def get_cabal_requirements(requirements):
     return _get_requirements(requirements, '==')
 
+def get_languages(bears):
+    language_dict = {}
+    for bear in bears:
+        language_dict[str(bear.name)] = list(sorted(bear.LANGUAGES))
+    for value in language_dict.values():
+        for n,i in enumerate(value):
+            if(i=='All'):
+                value[n]=True
+        if 'default' in value:
+            value.remove('default')
```

Comment:
@ayan-b , this is about `DocumentationStyleBear` which uses `DocstyleDefinition.get_available_definitions` - i think you were looking at this .. ?  It was caused by something in coalib ...?

Did you create an issue about it?

c.f. https://github.com/coala/coala-bears/pull/2688#discussion_r225751919